python/lsst/pipe/analysis/visitAnalysis.py

```diff
@@ -1133,4 +1339,11 @@ def calibrateCatalogs(self, dataRef, catalog, fluxMag0, photoCalibDataset, doApp
             zpLabel = "FLUXMAG0"
             calibrated = calibrateSourceCatalog(catalog, self.zp)
 
+        if doApplyExternalSkyWcs:
+            wcs = dataRef.get(repoInfo.skyWcsDataset)
+            for record in catalog:
+                record.updateCoord(wcs)
```

Comment:
This can be done vectorized (and faster!) with `afwTable.updateSourceCoords(wcs, catalog)`, see e.g. https://github.com/lsst/validate_drp/blob/7546aab731893bfa50f0c0f189572c2f86b2591b/python/lsst/validate/drp/matchreduce.py#L318

Reply:
Awesome...I like faster!  Thanks for the tip 👍